blog/dev_hackathon_v4_release.md

```diff
@@ -0,0 +1,134 @@
+(version_four_release)=
+# Dev Sprint: Getting the PyMC v4.0 Release Out
+
+:::{post} Jun 24, 2022
+:tags: sprint, contributing
+:category: news
+:author: Reshama Shaikh, Thomas Wiecki
+:image: 0
+:exclude:
+
+The PyMC team coordinated a development hackathon in May/June 2022 to complete updates and documentation in order to release [v4.0 of the PyMC library](https://twitter.com/pymc_devs/status/1533863780601446401).
+:::
+
+![v4 release social media card](../_static/hackathon_v4_release/v4release_card.png)
+
+The [v4.0 release of the PyMC library](https://twitter.com/pymc_devs/status/1533863780601446401) was officially announced on June 6, 2022.  To achieve this milestone, the PyMC team planned a week-long online sprint beginning May 30, 2022.  The goal was to complete remaining issues on the v4.0 release.  But, *mainly* it was to finalize the documentation and update the website for PyMC.
+
+There are 3 separate websites for [PyMC](https://www.pymc.io/welcome.html):  
+- [Website and Blog](https://www.pymc.io/welcome.html)
+- [Documentation (API)](https://www.pymc.io/projects/docs/en/stable/api.html)
+- [Examples](https://www.pymc.io/projects/examples/en/latest/gallery.html)
```

Comment:
```suggestion
```

Reply:
@OriolAbril 
Deleting the section on the website.  If you would like to add screenshots to explain the work to the website, feel free to do it.  Or the one sentence might be sufficient:
```txt
But, *mainly* it was to finalize the documentation and update the website for PyMC.
```